kai_mcp_solution_server/tests/test_multiple_integration.py

```
@@ -353,3 +359,183 @@
+        os.environ["KAI_LLM_PARAMS"] = json.dumps(
+            {
+                "model": "fake",
+                "responses": [
+                    f"You should add a smiley face to the file.",
+                ],
+            }
+        )
+
+        def stream_output(process: subprocess.Popen) -> None:
+            try:
+                assert process.stdout is not None
```

Ruff (0.12.2)

Use of `assert` detected

(S101)